When I use `@NotEmpty` to mark the `@QueryParam`, the request timeout.

### Expected behavior
request normal.

### Actual behavior
client timeout.
```
java.util.concurrent.ExecutionException: java.net.SocketTimeoutException: Request: http://127.0.0.1:53867/integration/test/validation/param exceeds read timeout: 6000ms

	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1908)
	at io.esastack.restlight.integration.jaxrs.test.ValidationTest.testParam(ValidationTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.net.SocketTimeoutException: Request: http://127.0.0.1:53867/integration/test/validation/param exceeds read timeout: 6000ms
	at io.esastack.httpclient.core.netty.ReadTimeoutTask.lambda$run$76(ReadTimeoutTask.java:49)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:750)
```

### Steps to reproduce
1.define the resource like below.
```java
@Controller
@Path("/validation/")
public class ValidationResource {

    @GET
    @Path("param")
    public UserData param(@NotEmpty @QueryParam("name") String name) {
        return UserData.Builder.anUserData()
                .name(name).build();
    }
}
```
2.request it like below.
```java
RestResponseBase responseBase = restClient.get(domain + "/validation/param").execute().toCompletableFuture().get();
Assert.assertEquals(HttpStatus.INTERNAL_SERVER_ERROR.code(), responseBase.status());
```

### Env
- Restlight version: 1.0.0-SNAPSHOT
- JVM version: 1.8
- OS: windows 10

--- a/restlight-ext/restlight-ext-validator/src/main/java/io/esastack/restlight/ext/validator/ConstraintDetail.java
+++ b/restlight-ext/restlight-ext-validator/src/main/java/io/esastack/restlight/ext/validator/ConstraintDetail.java
@@ -18,10 +18,10 @@ package io.esastack.restlight.ext.validator;
 public class ConstraintDetail {
 
     private final String property;
-    private final String invalidValue;
+    private final Object invalidValue;
     private final String message;
 
-    public ConstraintDetail(String property, String invalidValue, String message) {
+    public ConstraintDetail(String property, Object invalidValue, String message) {
         this.property = property;
         this.invalidValue = invalidValue;
         this.message = message;
@@ -31,7 +31,7 @@ public class ConstraintDetail {
         return property;
     }
 
-    public String getInvalidValue() {
+    public Object getInvalidValue() {
         return invalidValue;
     }
 
@@ -41,12 +41,10 @@ public class ConstraintDetail {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder("ConstraintDetail{");
-        sb.append("property='").append(property).append('\'');
-        sb.append(", invalidValue='").append(invalidValue).append('\'');
-        sb.append(", message='").append(message).append('\'');
-        sb.append('}');
-        return sb.toString();
+        return "ConstraintDetail{" + "property='" + property + '\'' +
+                ", invalidValue='" + invalidValue + '\'' +
+                ", message='" + message + '\'' +
+                '}';
     }
 }
 

--- a/restlight-ext/restlight-ext-validator/src/main/java/io/esastack/restlight/ext/validator/ValidationExceptionHandlerFactory.java
+++ b/restlight-ext/restlight-ext-validator/src/main/java/io/esastack/restlight/ext/validator/ValidationExceptionHandlerFactory.java
@@ -63,14 +63,17 @@ public class ValidationExceptionHandlerFactory implements ExceptionHandlerFactor
                         List<ConstraintDetail> details = InternalThreadLocalMap.get().arrayList(constraints.size());
                         for (ConstraintViolation<?> c : constraints) {
                             details.add(new ConstraintDetail(c.getPropertyPath().toString(),
-                                    c.getInvalidValue().toString(), c.getMessage()));
+                                    c.getInvalidValue(), c.getMessage()));
                         }
-                        context.response().entity(new ErrorDetail<>(context.request().path(), details.toString()));
+                        context.response().entity(new ErrorDetail<>(context.request().path(), details));
                     }
                     handled.complete(null);
                 } else {
                     handled.completeExceptionally(ex);
                 }
+            }).exceptionally(ex -> {
+                handled.completeExceptionally(ex);
+                return null;
             });
             return handled;
         }

